e2e/cypress/integration/account_settings/general/fullname_spec.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Account Settings > Sidebar > General', () => {
+    before(() => {
+        cy.apiLogin('user-1');
+
+        cy.visit('/');
+        cy.get('#channel_view').should('be.visible');
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+        cy.get('#accountSettings').should('be.visible').click();
+        cy.get('#accountSettingsModal').should('be.visible');
```

Comment:
Above lines can be removed.  It's just the same steps done by `cy.toAccountSettingsModal('user-1');`